src/commands/Fun/Random/roll.ts

```diff
@@ -92,13 +223,16 @@ export default class extends SteveCommand {
 		if (results.length === 1) {
 			const result = results[0];
 			const emoji = getEmoji(result.spec);
-			const message = `${emoji} You rolled: \`${result.rolls.join(', ')}\` ${emoji}`;
+			// just display the message when a single roll is made
+			const message = `${emoji} You rolled: ${result.message} ${emoji}`;
```

Comment:
needs to be internationalized as well

Reply:
A few lines below needed updating too. Should be fixed  with checks passing in ae7df243c747f41e68ae580918f4f94dffbf01a1

Reply:
Tested and working